src/server.c

```diff
@@ -5824,7 +5840,9 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "total_blocking_keys_on_nokey:%lu\r\n", blocking_keys_on_nokey,
                 "paused_reason:%s\r\n", paused_reason,
                 "paused_actions:%s\r\n", paused_actions,
-                "paused_timeout_milliseconds:%lld\r\n", paused_timeout));
+                "paused_timeout_milliseconds:%lld\r\n", paused_timeout,
+                "total_tunnel_sessions:%lld\r\n", server.stat_tunnel_sessions,
+                "active_tunnel_sessions:%lld\r\n", server.stat_active_tunnel_sessions));
```

Comment:
wouldn't a more appropriate name be tunnel clients? 

I also generally don't really like the term tunnel, these feel more like proxies than tunnels, but I don't feel that strongly about the name.